bool reset
---
bool reset_status
